docs/run-basic.md

```diff
@@ -0,0 +1,93 @@
+# Running the OMS Locally
+
+Follow these instructions to run the OMS locally, using the 
+Temporal Service provided that the `temporal` command. This 
+is an expanded version of the instructions found in the 
+_Quickstart_ section of the top-level README file.
+
+
+### Start the Temporal Service
+
+Run the following command in your terminal to start the Temporal 
```

Comment:
```suggestion
Run the following command in your terminal (from the root directory of your repo clone) to start the Temporal 
```

Reply:
Fixed in https://github.com/temporalio/reference-app-orders-go/commit/cbec91754ee86b51c3eb9c3cb1839e68893649ca